index.html

```diff
@@ -0,0 +1,858 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/9875f697fa.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>NEWYORKTIMES</title>
+  </head>
+
+  <body>
+    <!--NAVIGATION MANU-->
+    <div class="topnav">
+      <div class="nav-left">
+        <a href="#"><i class="fas fa-bars fa-sm"></i></a>
+        <a href="#"><i class="fas fa-search fa-sm"></i></a>
+        <a href="#space&cosmos">SPACE & COSMOS</a>
+      </div>
+
+      <a href="#"
+        ><img
+          src="./images/image2.png"
+          alt="New York times"
+          class="newyork-logo"
+      /></a>
+
+      <div class="nav-right">
+        <button type="submit" class="btn info">SUBSCRIBE NOW</button>
+        <button type="submit" class="btn info">LOG IN</button>
+      </div>
+    </div>
```

Comment:
The navigation bar is not well styles to look similar to the original space.
The test for login and subscribe buttons should be well aligned at the center. The icons still look much different together with Space & Cosmos heading.
Clone Page             |  Original Page
:-------------------------:|:-------------------------:
(![Screenshot from 2021-02-18 18-30-08](https://user-images.githubusercontent.com/44072711/108435872-5ffb0500-7218-11eb-8408-0e70c1476ad4.png)  | ![Screenshot from 2021-02-18 18-30-03](https://user-images.githubusercontent.com/44072711/108435894-68ebd680-7218-11eb-94f5-d7cfa002afe7.png)

Reply:
I am confused  what you mean I have worked on the nav bar alignment already, and the screenshot you are showing is of old version not current version with changed i made